booking/src/main/java/ru/org.dsr/domain/PackSearch.java

```diff
@@ -0,0 +1,24 @@
+package ru.org.dsr.domain;
+
+import ru.org.dsr.search.Search;
+import sun.awt.image.ImageWatched;
+
+import java.util.LinkedList;
+
+public class PackSearch {
+    private LinkedList<Search> searches = new LinkedList<>();
```

Comment:
Use just interface or base class in left part, just `List<Search>`. No need to stick to `LinkedList`, for some cases it could be `ArrayList`